lib/galaxy/datatypes/binary.py

```diff
@@ -1756,6 +1756,37 @@ def sniff(self, filename):
             return False
 
 
+class DAA(Binary):
+    """
+    Class describing an DAA (diamond alignment archive) file
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname( 'diamond.daa' )
+    >>> DAA().sniff( fname )
+    True
+    >>> fname = get_test_fname( 'interval.interval' )
+    >>> DMND().sniff( fname )
+    False
+    """
+    file_ext = "dmnd"
+    edam_format = ""
+
+    def __init__(self, **kwd):
+        Binary.__init__(self, **kwd)
+                                          
```

Comment:
The linter complains about the spaces on this line

Reply:
okay